docs/conf.py

```diff
@@ -35,8 +35,6 @@ def noop_decorator(*args, **kwargs):
     sys.modules["cudf"].DataFrame = type("DataFrame", (object,), {})
 if not hasattr(sys.modules["cupy"], "ndarray"):
     sys.modules["cupy"].ndarray = type("ndarray", (object,), {})
-if not hasattr(sys.modules["pyhdk"], "PyDbEngine"):
-    sys.modules["pyhdk"].PyDbEngine = type("PyDbEngine", (object,), {})
```

Comment:
This is deprecated and is not used anywhere.

Reply:
My understanding that it was kept for backward compatibility, maybe @AndreyPavlenko knows more

Reply:
I think it could be safely removed.